conformance-test/test-bench-util.ts

```diff
@@ -21,7 +21,7 @@ const PORT = new URL(HOST).port;
 const CONTAINER_NAME = 'storage-testbench';
 const DEFAULT_IMAGE_NAME =
   'gcr.io/cloud-devrel-public-resources/storage-testbench';
-const DEFAULT_IMAGE_TAG = 'v0.10.0';
+const DEFAULT_IMAGE_TAG = 'v0.11.0';
```

Comment:
We will need to be more aware when test bench releases and make sure we update this accordingly after locally testing. I am hesitant to go back to using `latest` after it broke our tests and potentially blocked releases for a few days.